pkg/utils/options/options.go

```diff
@@ -77,7 +78,7 @@ func New() *Options {
 	f.StringVar(&opts.AWSNodeNameConvention, "aws-node-name-convention", env.WithDefaultString("AWS_NODE_NAME_CONVENTION", string(IPName)), "The node naming convention used by the AWS cloud provider. DEPRECATION WARNING: this field may be deprecated at any time")
 	f.BoolVar(&opts.AWSENILimitedPodDensity, "aws-eni-limited-pod-density", env.WithDefaultBool("AWS_ENI_LIMITED_POD_DENSITY", true), "Indicates whether new nodes should use ENI-based pod density. DEPRECATED: Use `.spec.kubeletConfiguration.maxPods` to set pod density on a per-provisioner basis")
 	f.StringVar(&opts.AWSDefaultInstanceProfile, "aws-default-instance-profile", env.WithDefaultString("AWS_DEFAULT_INSTANCE_PROFILE", ""), "The default instance profile to use when provisioning nodes in AWS")
-	f.BoolVar(&opts.AWSEnablePodENI, "aws-enable-pod-eni", env.WithDefaultBool("AWS_ENABLE_POD_ENI", false), "If true then instances that support pod ENI will report a vpc.amazonaws.com/pod-eni resource")
+	f.BoolVar(&opts.AWSEnablePodENI, "aws-enable-pod-eni", env.WithDefaultBool("AWS_ENABLE_POD_ENI", false), fmt.Sprintf("If true then instances that support pod ENI will report a vpc.amazonaws.com/pod-eni resource. Refer to Karpenter Pod ENI documentation before enabling: https://karpenter.sh/%s/aws/provisioning/#pod-eni-security-groups-for-pods", project.GetReleaseVersion()))
```

Comment:
Interested to hear what people think about putting the version in the help message.  I could instead just refer to the "latest" docs page, which would be `https://karpenter.sh/docs/`.

Reply:
I think referring to latest might be easiest. Makes us have to call less functions.

Reply:
Yeah, that makes sense to me also.  I think the only caveat is that `docs` refers to the latest version of documentation.  Which, if the docs page is new, will only be available in `preview` until the next release.  Essentially the link will be broken until the next stable release.

Reply:
I updated the link